web/srv/handlers.go

```diff
@@ -55,6 +55,11 @@ func (h *handler) handleIndex(w http.ResponseWriter, req *http.Request, p httpro
 	if err != nil {
 		params.Error = true
 		params.ErrorMessage = err.Error()
+		params.Data = &pb.VersionInfo{
+			GoVersion:      "Unavailable",
+			BuildDate:      "Unavailable",
+			ReleaseVersion: "Unavailable",
```

Comment:
Is there value in making these strings constants? There may even be a string for this already

Reply:
I did a quick search through the code base and did not find anything useful for this.
I'm not sure if making these consts have a great value but will do so if you think there is.